torch_xla/csrc/debug_util.cpp

```diff
@@ -53,15 +53,20 @@ DebugUtil::GraphFormat DebugUtil::GetDefaultGraphFormat() {
 
 std::string DebugUtil::GetTensorsGraphInfo(
     absl::Span<const XLATensorPtr> tensors, const std::vector<size_t>* indices,
```

Comment:
we don't really need `tensors ` and `indices` if we always pass`ir_values`, let's just clean this up

Reply:
The only reason for having tensors is - we get device from it